app/helpers/tax_return_status_helper.rb

```diff
@@ -0,0 +1,28 @@
+module TaxReturnStatusHelper
+
+
+  def grouped_tax_return_status_options
+    options = []
+    TaxReturn::STAGES.map do |stage|
+      statuses = TaxReturn.statuses_for(stage)
+      select = [translated_stage(stage), status_options = []]
+      statuses.map do |status, _|
+        status_options.push([translated_status(status), status])
+      end
+      options << select
+    end
+    options
+  end
+
+  def translated_status(status)
+    I18n.t("case_management.tax_returns.status.#{status}")
```

Comment:
Constructing the translation string like this ensures that all of the status translations are marked as used, and same with the stage helper. In order to better ensure that we really have translations, I added some tests for checking for translations in the tax_return model.